holonote/annotate/display.py

```diff
@@ -425,13 +432,24 @@ def get_indices_by_position(self, **inputs) -> list[Any]:
             iter_mask = (
                 (df[f"start[{k}]"] <= v) & (v < df[f"end[{k}]"]) for k, v in inputs.items()
             )
+            subset = reduce(np.logical_and, iter_mask)
+            out = list(df[subset].index)
+        elif self.region_format == "point-point":
+            xk, yk = list(inputs.keys())
+            distance = (
+                (df[f"point[{xk}]"] - inputs[xk]) ** 2 + (df[f"point[{yk}]"] - inputs[yk]) ** 2
+            ) ** 0.5
+            if (distance > self.nearest_2d_point_threshold).all():
```

Comment:
```suggestion
            )
            if (distance > self.nearest_2d_point_threshold ** 2).all():
```

If we want to squeeze out the sweet (microsecond) performance.